src/cmds/Makefile.am

```diff
@@ -86,6 +86,10 @@ dist_bin_SCRIPTS = \
 	pbsrun_unwrap \
 	pbsrun_wrap
 
+common_cflags = \
+	-I$(top_srcdir)/src/include \
+	@KRB5_CFLAGS@
```

Comment:
This is kerberos specific. Since our goal is to make the core code free of specific authentication/encryption calls, this should also be made generic. @vchlum perhaps we need your help in this area. Also we might need help is carving out the other kerberos specific functionality like ticket renewal etc into a kerberos specific library.

Reply:
Sure @subhasisb, I am willing to help with that. What is your intention - do you want to refactor it as a part of this PR?

Reply:
hi @vchlum - thanks. I did not mean we need to do as part of this PR - a separate PR is also fine. The point is, how do  we take out the individual authentication related code from the core of PBS and make it part of a library - it may even need an extension to the proposed libAuth library interface.